ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_shared_utils/platform.py

```diff
@@ -41,7 +45,7 @@
         "feature_support": {"queue_monitor_length_notify": False, "poe": True},
     },
     {
-        "platforms": ["750", "755", "758"],
+        "platforms": ["750", "755", "758", "720D", "722XP", "710P"],
```

Comment:
Think we need to change `"720D"` to `"720DP"` since the 720DTs don't support POE

Reply:
I agree with @jrecchia1029 - see platform specification: https://www.arista.com/en/products/720d-series/specifications

Reply:
Thanks.  Will update this shortly.